doc/main.dox

```diff
@@ -1,14 +1,12 @@
 namespace lsst { namespace utils {
 
 /**
-\mainpage lsst::utils;  utility classes
+\mainpage lsst::cpputils;  utility classes
 
 \section utils_introduction Introduction
 
-This is a collection of miscellaneous utility functions, classes and macros, including:
-- lsst/p_lsstSwig.i useful macros for SWIG interface files
-- \ref lsst.utils.tests utilties to aid in unit testing
-- \ref lsst::utils::getPackageDir and lsst.utils.getPackageDir provide an equivalent
+This is a collection of miscellaneous C++-based utility functions, classes and macros, including:
+- \ref lsst::utils::getPackageDir and lsst.cpputils.getPackageDir provide an equivalent
```

Comment:
Isn't this messed up?  Since this is doxygen, it should only mention `lsst::utils::getPackageDir` from here.  It ought to mention the other things that are still here as well.  And no one should be referring to `lsst.cpputils.getPackagDir`; in fact, that shouldn't even exist, so I think you can get rid of `python/lsst/cpputils/_packaging.cc`.